src/Escaper/EscaperInternal.h

```diff
@@ -0,0 +1,32 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ESCAPER_INTERNAL_H
+#define ESCAPER_INTERNAL_H
+
+#define PACKET_MAX_SIZE 8 * 1024
```

Comment:
I think that this shouldn't be defined here. This shall be a parameter for Escaper. (For IP the buffer is 8k, but for SAMV71 serial is shall be something else (buffer size required by HAL I guess).

Reply:
What do you suggest, then? Should the size of this array be parameter passed during compilation?

Reply:
Its done.